### ISSUE TITLE
Can i make a logo?

### ISSUE THREAD

**Issue by debpedrano** [codepath/android_guides#340]
Hello, can I make a logo for your project? I just want to practice my skill in graphics designing.

**Comment by jitendra1607** [comment]
Sure i you can in return of credits mentioned.

**Comment by debpedrano** [comment]
Great, @jitendra1607 I will comment my design right after I am done.

**Comment by jitendra1607** [comment]
This can be inspiration.

**Comment by debpedrano** [comment]
I hope I did it well, those three on the top are the logo. 

![image](https://user-images.githubusercontent.com/40510200/46258589-00451780-c500-11e8-95ce-a2dd4b0f93c7.png)

**Comment by sbtgE** [comment]
Yep sure. I am interested to go with you if you are looking for experience.
Are you still in?

**Comment by debpedrano** [comment]
I am in on that @sbtgE .

**Comment by debpedrano** [comment]
should I make a pull request now? about the project logo?

**Comment by debpedrano** [comment]
Hello, can I update the issue? please response. Thank you :) @jitendra1607
